[qtbase] Build error on x64-linux

Package: qtbase[brotli,concurrent,core,dbus,default-features,doubleconversion,egl,fontconfig,freetype,gui,harfbuzz,icu,jpeg,network,opengl,openssl,pcre2,png,sql,sql-psql,sql-sqlite,testlib,thread,widgets,xcb,xcb-xlib,xkb,xkbcommon-x11,xlib,xrender,zstd]:x64-linux@6.6.1#3

**Host Environment**

- Host: x64-linux
- Compiler: GNU 11.4.0
-    vcpkg-tool version: 2023-12-12-1c9ec1978a6b0c2b39c9e9554a96e3e275f7556e
    vcpkg-scripts version: 4874bea8e 2023-12-20 (12 hours ago)

**To Reproduce**

`vcpkg install quazip:x64-linux`
**Failure logs**

```
CMake Warning at ports/qtbase/portfile.cmake:39 (message):
  qtbase currently requires packages from the system package manager.  They
  can be installed on Ubuntu systems via sudo apt-get install '^libxcb.*-dev'
  libx11-xcb-dev libglu1-mesa-dev libxrender-dev libxi-dev libxkbcommon-dev
  libxkbcommon-x11-dev.
Call Stack (most recent call first):
  scripts/ports.cmake:170 (include)


-- Using cached qtbase-everywhere-src-6.6.1.tar.xz.
-- Cleaning sources at /home/charlie/Documents/workspaces/workspace-vscode/vcpkg/buildtrees/qtbase/src/here-src-6-8b9bc1db6f.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source /home/charlie/Documents/workspaces/workspace-vscode/vcpkg/downloads/qtbase-everywhere-src-6.6.1.tar.xz
-- Applying patch allow_outside_prefix.patch
-- Applying patch config_install.patch
-- Applying patch fix_cmake_build.patch
-- Applying patch fix_cmake_build_type.patch
-- Applying patch harfbuzz.patch
-- Applying patch fix_egl.patch
-- Applying patch fix_egl_2.patch
-- Applying patch installed_dir.patch
-- Applying patch GLIB2-static.patch
-- Applying patch clang-cl_source_location.patch
-- Applying patch clang-cl_QGADGET_fix.diff
-- Applying patch fix-host-aliasing.patch
-- Applying patch dont_force_cmakecache_latest.patch
-- Using source at /home/charlie/Documents/workspaces/workspace-vscode/vcpkg/buildtrees/qtbase/src/here-src-6-8b9bc1db6f.clean
-- Configuring x64-linux
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:112 (message):
    Command failed: /home/charlie/Documents/workspaces/workspace-vscode/vcpkg/downloads/tools/ninja/1.10.2-linux/ninja -v
    Working Directory: /home/charlie/Documents/workspaces/workspace-vscode/vcpkg/buildtrees/qtbase/x64-linux-rel/vcpkg-parallel-configure
    Error code: 1
    See logs for more information:
      /home/charlie/Documents/workspaces/workspace-vscode/vcpkg/buildtrees/qtbase/config-x64-linux-dbg-CMakeCache.txt.log
      /home/charlie/Documents/workspaces/workspace-vscode/vcpkg/buildtrees/qtbase/config-x64-linux-rel-CMakeCache.txt.log
      /home/charlie/Documents/workspaces/workspace-vscode/vcpkg/buildtrees/qtbase/config-x64-linux-out.log

Call Stack (most recent call first):
  installed/x64-linux/share/vcpkg-cmake/vcpkg_cmake_configure.cmake:252 (vcpkg_execute_required_process)
  ports/qtbase/cmake/qt_install_submodule.cmake:147 (vcpkg_cmake_configure)
  ports/qtbase/cmake/qt_install_submodule.cmake:341 (qt_cmake_configure)
  ports/qtbase/portfile.cmake:293 (qt_install_submodule)
  scripts/ports.cmake:170 (include)



```
<details><summary>/home/charlie/Documents/workspaces/workspace-vscode/vcpkg/buildtrees/qtbase/config-x64-linux-out.log</summary>

```
[1/2] "/home/charlie/Documents/workspaces/workspace-vscode/vcpkg/downloads/tools/cmake-3.27.1-linux/cmake-3.27.1-linux-x86_64/bin/cmake" -E chdir "../../x64-linux-dbg" "/home/charlie/Documents/workspaces/workspace-vscode/vcpkg/downloads/tools/cmake-3.27.1-linux/cmake-3.27.1-linux-x86_64/bin/cmake" "/home/charlie/Documents/workspaces/workspace-vscode/vcpkg/buildtrees/qtbase/src/here-src-6-8b9bc1db6f.clean" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Debug" "-DCMAKE_INSTALL_PREFIX=/home/charlie/Documents/workspaces/workspace-vscode/vcpkg/packages/qtbase_x64-linux/debug" "-DFETCHCONTENT_FULLY_DISCONNECTED=ON" "-DQT_USE_DEFAULT_CMAKE_OPTIMIZATION_FLAGS:BOOL=ON" "-DCMAKE_FIND_PACKAGE_TARGETS_GLOBAL=ON" "-DQT_BUILD_EXAMPLES:BOOL=OFF" "-DQT_BUILD_TESTS:BOOL=OFF" "-DQT_BUILD_BENCHMARKS:BOOL=OFF" "-DHOST_PERL:PATH=/usr/bin/perl" "-DINSTALL_BINDIR:STRING=bin" "-DINSTALL_LIBEXECDIR:STRING=bin" "-DINSTALL_PLUGINSDIR:STRING=Qt6/plugins" "-DINSTALL_QMLDIR:STRING=Qt6/qml" "-DFEATURE_appstore_compliant=OFF" "-DFEATURE_zstd=ON" "-DFEATURE_framework=OFF" "-DFEATURE_concurrent=ON" "-DFEATURE_future=ON" "-DFEATURE_dbus=ON" "-DFEATURE_gui=ON" "-DFEATURE_thread=ON" "-DFEATURE_network=ON" "-DFEATURE_sql=ON" "-DFEATURE_widgets=ON" "-DFEATURE_testlib=ON" "-DCMAKE_REQUIRE_FIND_PACKAGE_zstd=ON" "-DCMAKE_DISABLE_FIND_PACKAGE_ZSTD=OFF" "-DCMAKE_DISABLE_FIND_PACKAGE_WrapDBus1=OFF" "-DCMAKE_DISABLE_FIND_PACKAGE_Libudev:BOOL=ON" "-DFEATURE_xml:BOOL=ON" "-DFEATURE_doubleconversion=ON" "-DFEATURE_glib=OFF" "-DFEATURE_icu=ON" "-DFEATURE_pcre2=ON" "-DCMAKE_REQUIRE_FIND_PACKAGE_GLIB2=OFF" "-DCMAKE_DISABLE_FIND_PACKAGE_ICU=OFF" "-DCMAKE_DISABLE_FIND_PACKAGE_GLIB2=ON" "-DCMAKE_DISABLE_FIND_PACKAGE_LTTngUST:BOOL=ON" "-DCMAKE_DISABLE_FIND_PACKAGE_PPS:BOOL=ON" "-DCMAKE_DISABLE_FIND_PACKAGE_Slog2:BOOL=ON" "-DCMAKE_DISABLE_FIND_PACKAGE_Libsystemd:BOOL=ON" "-DCMAKE_DISABLE_FIND_PACKAGE_WrapBacktrace:BOOL=ON" "-DCMAKE_DISABLE_FIND_PACKAGE_PPS:BOOL=ON" "-DCMAKE_DISABLE_FIND_PACKAGE_Slog2:BOOL=ON" "-DFEATURE_openssl=ON" "-DFEATURE_brotli=ON" "-DCMAKE_DISABLE_FIND_PACKAGE_WrapBrotli=OFF" "-DCMAKE_DISABLE_FIND_PACKAGE_WrapOpenSSL=OFF" "-DINPUT_openssl=linked" "-DCMAKE_DISABLE_FIND_PACKAGE_Libproxy:BOOL=ON" "-DCMAKE_DISABLE_FIND_PACKAGE_GSSAPI:BOOL=ON" "-DCMAKE_DISABLE_FIND_PACKAGE_WrapResolv:BOOL=ON" "-DFEATURE_freetype=ON" "-DFEATURE_harfbuzz=ON" "-DFEATURE_fontconfig=ON" "-DFEATURE_jpeg=ON" "-DFEATURE_png=ON" "-DFEATURE_opengl=ON" "-DFEATURE_xlib=ON" "-DFEATURE_xkbcommon=ON" "-DFEATURE_xcb=ON" "-DFEATURE_xcb_xlib=ON" "-DFEATURE_xkbcommon_x11=ON" "-DFEATURE_xrender=ON" "-DFEATURE_xcb_native_painting=ON" "-DFEATURE_opengles2=OFF" "-DFEATURE_egl=ON" "-DCMAKE_DISABLE_FIND_PACKAGE_Vulkan=ON" "-DCMAKE_DISABLE_FIND_PACKAGE_WrapOpenGL=OFF" "-DCMAKE_DISABLE_FIND_PACKAGE_EGL=OFF" "-DCMAKE_DISABLE_FIND_PACKAGE_GLESv2=ON" "-DCMAKE_DISABLE_FIND_PACKAGE_Fontconfig=OFF" "-DCMAKE_DISABLE_FIND_PACKAGE_WrapSystemHarfbuzz=OFF" "-DCMAKE_DISABLE_FIND_PACKAGE_JPEG=OFF" "-DCMAKE_DISABLE_FIND_PACKAGE_PNG=OFF" "-DCMAKE_DISABLE_FIND_PACKAGE_X11=OFF" "-DCMAKE_DISABLE_FIND_PACKAGE_XKB=OFF" "-DCMAKE_DISABLE_FIND_PACKAGE_XCB=OFF" "-DCMAKE_DISABLE_FIND_PACKAGE_X11_XCB=OFF" "-DCMAKE_DISABLE_FIND_PACKAGE_XKB_COMMON_X11=OFF" "-DCMAKE_DISABLE_FIND_PACKAGE_XRender=OFF" "-DINPUT_xcb=yes" "-DINPUT_xkbcommon=yes" "-DFEATURE_opengles3:BOOL=OFF" "-DFEATURE_opengles31:BOOL=OFF" "-DFEATURE_opengles32:BOOL=OFF" "-DCMAKE_DISABLE_FIND_PACKAGE_ATSPI2:BOOL=ON" "-DCMAKE_DISABLE_FIND_PACKAGE_DirectFB:BOOL=ON" "-DCMAKE_DISABLE_FIND_PACKAGE_Libdrm:BOOL=ON" "-DCMAKE_DISABLE_FIND_PACKAGE_gbm:BOOL=ON" "-DCMAKE_DISABLE_FIND_PACKAGE_Libinput:BOOL=ON" "-DCMAKE_DISABLE_FIND_PACKAGE_Mtdev:BOOL=ON" "-DCMAKE_DISABLE_FIND_PACKAGE_Tslib:BOOL=ON" "-DFEATURE_system_sqlite=ON" "-DFEATURE_sql_odbc=OFF" "-DFEATURE_sql_mysql=OFF" "-DFEATURE_sql_oci=OFF" "-DCMAKE_DISABLE_FIND_PACKAGE_PostgreSQL=OFF" "-DCMAKE_DISABLE_FIND_PACKAGE_SQLite3=OFF" "-DCMAKE_DISABLE_FIND_PACKAGE_ODBC=ON" "-DCMAKE_DISABLE_FIND_PACKAGE_MySQL=ON" "-DCMAKE_DISABLE_FIND_PACKAGE_Oracle=ON" "-DCMAKE_DISABLE_FIND_PACKAGE_DB2:BOOL=ON" "-DCMAKE_DISABLE_FIND_PACKAGE_Interbase:BOOL=ON" "-DCMAKE_DISABLE_FIND_PACKAGE_Mimer:BOOL=ON" "-DCMAKE_DISABLE_FIND_PACKAGE_CUPS:BOOL=ON" "-DFEATURE_gtk3=OFF" "-DCMAKE_DISABLE_FIND_PACKAGE_GTK3=ON" "-DINPUT_doubleconversion:STRING=system" "-DINPUT_freetype:STRING=system" "-DINPUT_harfbuzz:STRING=system" "-DINPUT_libb2:STRING=system" "-DINPUT_libjpeg:STRING=system" "-DINPUT_libmd4c:STRING=qt" "-DINPUT_libpng:STRING=systemsystem" "-DQT_USE_BUNDLED_BundledFreetype:BOOL=FALSE" "-DQT_USE_BUNDLED_BundledHarfbuzz:BOOL=FALSE" "-DQT_USE_BUNDLED_BundledLibpng:BOOL=FALSE" "-DQT_USE_BUNDLED_BundledPcre2:BOOL=FALSE" "-DINPUT_bundled_xcb_xinput:STRING=no" "-DFEATURE_force_debug_info:BOOL=ON" "-DFEATURE_relocatable:BOOL=ON" "-DQT_NO_FORCE_SET_CMAKE_BUILD_TYPE:BOOL=ON" "-DCMAKE_MAKE_PROGRAM=/home/charlie/Documents/workspaces/workspace-vscode/vcpkg/downloads/tools/ninja/1.10.2-linux/ninja" "-DCMAKE_SYSTEM_NAME=Linux" "-DBUILD_SHARED_LIBS=OFF" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=/home/charlie/Documents/workspaces/workspace-vscode/vcpkg/scripts/toolchains/linux.cmake" "-DVCPKG_TARGET_TRIPLET=x64-linux" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=external" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=/home/charlie/Documents/workspaces/workspace-vscode/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=/home/charlie/Documents/workspaces/workspace-vscode/vcpkg" "-D_VCPKG_INSTALLED_DIR=/home/charlie/Documents/workspaces/workspace-vscode/vcpkg/installed" "-DVCPKG_MANIFEST_INSTALL=OFF" "-DINSTALL_DOCDIR:STRING=../doc/Qt6/" "-DINSTALL_INCLUDEDIR:STRING=../include/Qt6/" "-DINSTALL_TRANSLATIONSDIR:STRING=../translations/Qt6/" "-DINSTALL_DESCRIPTIONSDIR:STRING=../share/Qt6/modules" "-DINSTALL_MKSPECSDIR:STRING=../share/Qt6/mkspecs" "-DFEATURE_debug:BOOL=ON"
FAILED: ../../x64-linux-dbg/CMakeCache.txt 
...
Skipped 269 lines
...
-- The CXX compiler identification is GNU 11.4.0
-- The C compiler identification is GNU 11.4.0
-- The ASM compiler identification is GNU
-- Found assembler: /usr/bin/cc
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
            -DVCPKG_TARGET_TRIPLET=x64-linux
            -DVCPKG_TARGET_ARCHITECTURE=x64
            -DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=/home/charlie/Documents/workspaces/workspace-vscode/vcpkg/scripts/toolchains/linux.cmake
            -DZ_VCPKG_ROOT_DIR=/home/charlie/Documents/workspaces/workspace-vscode/vcpkg
            -DCMAKE_C_FLAGS=-fPIC
            -DCMAKE_C_FLAGS_DEBUG=-g
            -DCMAKE_C_FLAGS_RELEASE=-O3 -DNDEBUG
            -DCMAKE_C_FLAGS_RELWITHDEBINFO=-O2 -g -DNDEBUG
            -DCMAKE_CXX_FLAGS=-fPIC
            -DCMAKE_CXX_FLAGS_DEBUG=-g
            -DCMAKE_CXX_FLAGS_RELEASE=-O3 -DNDEBUG
            -DCMAKE_CXX_FLAGS_RELWITHDEBINFO=-O2 -g -DNDEBUG
            -DCMAKE_OBJCOPY=/usr/bin/objcopy
            -DCMAKE_TOOLCHAIN_FILE=/home/charlie/Documents/workspaces/workspace-vscode/vcpkg/scripts/buildsystems/vcpkg.cmake
            -DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=/home/charlie/Documents/workspaces/workspace-vscode/vcpkg/scripts/toolchains/linux.cmake
            -DCMAKE_C_STANDARD=11
            -DCMAKE_C_STANDARD_REQUIRED=ON
            -DCMAKE_CXX_STANDARD=17
            -DCMAKE_CXX_STANDARD_REQUIRED=ON
            -DCMAKE_MODULE_PATH:STRING=/home/charlie/Documents/workspaces/workspace-vscode/vcpkg/buildtrees/qtbase/src/here-src-6-8b9bc1db6f.clean/cmake/platforms
CMake Error at cmake/QtBuildInformation.cmake:508 (message):
  Feature "opengl": Forcing to "ON" breaks its condition:

      QT_FEATURE_opengl_desktop OR QT_FEATURE_opengl_dynamic OR QT_FEATURE_opengles2

  Condition values dump:

      QT_FEATURE_opengl_desktop = "OFF"
      QT_FEATURE_opengl_dynamic = "OFF"
      QT_FEATURE_opengles2 = "OFF"

Call Stack (most recent call first):
  cmake/QtFeature.cmake:318 (qt_configure_add_report_error)
  cmake/QtFeature.cmake:438 (qt_feature_check_and_save_internal_value)
  cmake/QtFeature.cmake:707 (qt_evaluate_feature)
  cmake/QtFeature.cmake:676 (qt_feature_module_end)
  src/CMakeLists.txt:13 (qt_feature_evaluate_features)



-- Configuration summary shown below. It has also been written to /home/charlie/Documents/workspaces/workspace-vscode/vcpkg/buildtrees/qtbase/x64-linux-rel/config.summary
-- Configure with --log-level=STATUS or higher to increase CMake's message verbosity. The log level does not persist across reconfigurations.
 
-- Configure summary:

Building for: linux-g++ (x86_64, CPU features: )
Compiler: gcc 11.4.0
Build options:
  Mode ................................... release
  Optimize release build for size ........ no
  Fully optimize release builds (-O3) .... no
  Building shared libraries .............. no
  Using ccache ........................... no
  Unity Build ............................ no
  Using new DTAGS ........................ yes
  Generating GDB index ................... no
  Relocatable ............................ yes
  Using precompiled headers .............. yes
  Using Link Time Optimization (LTCG) .... no
  Using Intel CET ........................ yes
  Target compiler supports:
    x86 Intrinsics ....................... Basic VAES AVX512VBMI2
  Sanitizers:
    Addresses ............................ no
    Threads .............................. no
    Memory ............................... no
    Fuzzer (instrumentation only) ........ no
    Undefined ............................ no
  Build parts ............................ libs tools
  Install examples sources ............... no
Qt modules and options:
  Qt Concurrent .......................... yes
  Qt D-Bus ............................... yes
  Qt D-Bus directly linked to libdbus .... yes
  Qt Gui ................................. yes
  Qt Network ............................. yes
  Qt PrintSupport ........................ yes
  Qt Sql ................................. yes
  Qt Testlib ............................. yes
  Qt Widgets ............................. yes
  Qt Xml ................................. yes
Support enabled for:
  Using pkg-config ....................... yes
  Using vcpkg ............................ yes
  udev ................................... no
  OpenSSL ................................ yes
    Qt directly linked to OpenSSL ........ yes
  OpenSSL 1.1 ............................ no
  OpenSSL 3.0 ............................ yes
  Using system zlib ...................... yes
  Zstandard support ...................... yes
  Thread support ......................... yes
Common build options:
  Linker can resolve circular dependencies  yes
Qt Core:
  backtrace .............................. no
  DoubleConversion ....................... yes
    Using system DoubleConversion ........ yes
  CLONE_PIDFD support in forkfd .......... yes
  GLib ................................... no
  ICU .................................... yes
  Using system libb2 ..................... yes
  Built-in copy of the MIME database ..... yes
  Application permissions ................ yes
  Defaulting legacy IPC to POSIX ......... no
  Tracing backend ........................ <none>
  OpenSSL based cryptographic hash ....... no
  Logging backends:
    journald ............................. no
    syslog ............................... no
    slog2 ................................ no
  PCRE2 .................................. yes
    Using system PCRE2 ................... yes
Qt Sql:
  SQL item models ........................ yes
Qt Network:
  getifaddrs() ........................... yes
  IPv6 ifname ............................ yes
  libproxy ............................... no
  Linux AF_NETLINK ....................... yes
  DTLS ................................... yes
  OCSP-stapling .......................... yes
  SCTP ................................... no
  Use system proxies ..................... yes
  GSSAPI ................................. no
  Brotli Decompression Support ........... yes
  qIsEffectiveTLD() ...................... yes
    Built-in publicsuffix database ....... yes
    System publicsuffix database ......... yes
Core tools:
  Android deployment tool ................ yes
  macOS deployment tool .................. no
  Windows deployment tool ................ no
  qmake .................................. yes
Qt Gui:
  Accessibility .......................... yes
  FreeType ............................... yes
    Using system FreeType ................ yes
  HarfBuzz ............................... yes
    Using system HarfBuzz ................ yes
  Fontconfig ............................. yes
  Image formats:
    GIF .................................. yes
    ICO .................................. yes
    JPEG ................................. yes
      Using system libjpeg ............... yes
    PNG .................................. yes
      Using system libpng ................ yes
  Text formats:
    HtmlParser ........................... yes
    CssParser ............................ yes
    OdfWriter ............................ yes
    MarkdownReader ....................... yes
      Using system libmd4c ............... no
    MarkdownWriter ....................... yes
  EGL .................................... yes
  OpenVG ................................. no
  OpenGL:
    Desktop OpenGL ....................... no
    OpenGL ES 2.0 ........................ no
    OpenGL ES 3.0 ........................ no
    OpenGL ES 3.1 ........................ no
    OpenGL ES 3.2 ........................ no
  Vulkan ................................. no
  Session Management ..................... yes
Features used by QPA backends:
  evdev .................................. yes
  libinput ............................... no
  HiRes wheel support in libinput ........ no
  INTEGRITY HID .......................... no
  mtdev .................................. no
  tslib .................................. no
  xkbcommon .............................. yes
  X11 specific:
    XLib ................................. yes
    XCB Xlib ............................. yes
    EGL on X11 ........................... yes
    xkbcommon-x11 ........................ yes
    xcb-sm ............................... yes
QPA backends:
  DirectFB ............................... no
  EGLFS .................................. yes
  EGLFS details:
    EGLFS OpenWFD ........................ no
    EGLFS i.Mx6 .......................... no
    EGLFS i.Mx6 Wayland .................. no
    EGLFS RCAR ........................... no
    EGLFS EGLDevice ...................... no
    EGLFS GBM ............................ no
    EGLFS VSP2 ........................... no
    EGLFS Mali ........................... no
    EGLFS Raspberry Pi ................... no
    EGLFS X11 ............................ yes
  LinuxFB ................................ yes
  VNC .................................... yes
  VK_KHR_display ......................... no
  QNX:
    lgmon ................................ no
    IMF .................................. no
  XCB:
    Using system-provided xcb-xinput ..... yes
    GL integrations:
      GLX Plugin ......................... yes
        XCB GLX .......................... yes
      EGL-X11 Plugin ..................... yes
  Windows:
    Direct 2D ............................ no
    Direct 2D 1.1 ........................ no
    DirectWrite .......................... no
    DirectWrite 3 ........................ no
Qt Widgets:
  GTK+ ................................... no
  Styles ................................. Fusion Windows
Qt Testlib:
  Tester for item models ................. yes
  Batch tests ............................ no
Qt PrintSupport:
  CUPS ................................... no
Qt Sql Drivers:
  DB2 (IBM) .............................. no
  InterBase .............................. no
  MySql .................................. no
  OCI (Oracle) ........................... no
  ODBC ................................... no
  PostgreSQL ............................. yes
  SQLite ................................. yes
    Using system provided SQLite ......... yes
  Mimer .................................. no
 

Note: Using static linking will disable the use of dynamically loaded plugins. Make sure to import all needed static plugins, or compile needed modules into the library.
Note: Disabling X11 Accessibility Bridge: D-Bus or AT-SPI is missing.

ERROR: Feature "opengl": Forcing to "ON" breaks its condition:
    QT_FEATURE_opengl_desktop OR QT_FEATURE_opengl_dynamic OR QT_FEATURE_opengles2
Condition values dump:
    QT_FEATURE_opengl_desktop = "OFF"
    QT_FEATURE_opengl_dynamic = "OFF"
    QT_FEATURE_opengles2 = "OFF"

ERROR: The OpenGL functionality tests failed! You might need to modify the OpenGL package search path by setting the OpenGL_DIR CMake variable to the OpenGL library's installation directory.

CMake Error at cmake/QtBuildInformation.cmake:194 (message):
  Check the configuration messages for an error that has occurred.
Call Stack (most recent call first):
  cmake/QtBuildInformation.cmake:24 (qt_configure_print_summary)
  cmake/QtBuildInternals/QtBuildInternalsConfig.cmake:593 (qt_print_feature_summary)
  CMakeLists.txt:211 (qt_build_repo_end)


-- Configuring incomplete, errors occurred!
ninja: build stopped: subcommand failed.
```
</details>

<details><summary>/home/charlie/Documents/workspaces/workspace-vscode/vcpkg/buildtrees/qtbase/config-x64-linux-rel-CMakeCache.txt.log</summary>

```
# This is the CMakeCache file.
# For build in directory: /home/charlie/Documents/workspaces/workspace-vscode/vcpkg/buildtrees/qtbase/x64-linux-rel
# It was generated by CMake: /home/charlie/Documents/workspaces/workspace-vscode/vcpkg/downloads/tools/cmake-3.27.1-linux/cmake-3.27.1-linux-x86_64/bin/cmake
# You can edit this file to change values found and used by cmake.
# If you do not want to change any of the values, simply exit the editor.
# If you do want to change a value, simply edit, save, and exit the editor.
# The syntax for the file is as follows:
# KEY:TYPE=VALUE
# KEY is the name of a variable in the cache.
# TYPE is a hint to GUIs for the type of VALUE, DO NOT EDIT TYPE!.
# VALUE is the current value for the KEY.

########################
# EXTERNAL cache entries
########################

//Path to a library.
BROTLICOMMON_LIBRARY_DEBUG:FILEPATH=/home/charlie/Documents/workspaces/workspace-vscode/vcpkg/installed/x64-linux/debug/lib/libbrotlicommon.a

//Path to a library.
BROTLICOMMON_LIBRARY_RELEASE:FILEPATH=/home/charlie/Documents/workspaces/workspace-vscode/vcpkg/installed/x64-linux/lib/libbrotlicommon.a

//Path to a library.
BROTLIDEC_LIBRARY_DEBUG:FILEPATH=/home/charlie/Documents/workspaces/workspace-vscode/vcpkg/installed/x64-linux/debug/lib/libbrotlidec.a

//Path to a library.
BROTLIDEC_LIBRARY_RELEASE:FILEPATH=/home/charlie/Documents/workspaces/workspace-vscode/vcpkg/installed/x64-linux/lib/libbrotlidec.a

//Build tests for the Qt build system
BUILD_CMAKE_TESTING:BOOL=OFF

//Build Qt statically or dynamically
BUILD_SHARED_LIBS:BOOL=OFF

//Build Qt using precompiled headers?
BUILD_WITH_PCH:BOOL=ON

//Path to a file.
BZIP2_INCLUDE_DIR:PATH=/home/charlie/Documents/workspaces/workspace-vscode/vcpkg/installed/x64-linux/include

//Path to a library.
BZIP2_LIBRARY_DEBUG:FILEPATH=/home/charlie/Documents/workspaces/workspace-vscode/vcpkg/installed/x64-linux/debug/lib/libbz2d.a

//Path to a library.
BZIP2_LIBRARY_RELEASE:FILEPATH=/home/charlie/Documents/workspaces/workspace-vscode/vcpkg/installed/x64-linux/lib/libbz2.a

//Path to a program.
CMAKE_ADDR2LINE:FILEPATH=/usr/bin/addr2line

//Path to a program.
CMAKE_AR:FILEPATH=/usr/bin/ar

//ASM compiler
CMAKE_ASM_COMPILER:FILEPATH=/usr/bin/cc

//A wrapper around 'ar' adding the appropriate '--plugin' option
// for the GCC compiler
CMAKE_ASM_COMPILER_AR:FILEPATH=/usr/bin/gcc-ar

//A wrapper around 'ranlib' adding the appropriate '--plugin' option
// for the GCC compiler
CMAKE_ASM_COMPILER_RANLIB:FILEPATH=/usr/bin/gcc-ranlib

//Flags used by the ASM compiler during all build types.
CMAKE_ASM_FLAGS:STRING=

//Flags used by the ASM compiler during DEBUG builds.
CMAKE_ASM_FLAGS_DEBUG:STRING=-g

//Flags used by the ASM compiler during MINSIZEREL builds.
CMAKE_ASM_FLAGS_MINSIZEREL:STRING=-Os -DNDEBUG

//Flags used by the ASM compiler during RELEASE builds.
CMAKE_ASM_FLAGS_RELEASE:STRING=-O3 -DNDEBUG

//Flags used by the ASM compiler during RELWITHDEBINFO builds.
CMAKE_ASM_FLAGS_RELWITHDEBINFO:STRING=-O2 -g -DNDEBUG

//Choose the type of build, options are: None Debug Release RelWithDebInfo
// MinSizeRel ...
CMAKE_BUILD_TYPE:STRING=Release

CMAKE_CROSSCOMPILING:BOOL=OFF

//CXX compiler
CMAKE_CXX_COMPILER:FILEPATH=/usr/bin/c++

//A wrapper around 'ar' adding the appropriate '--plugin' option
// for the GCC compiler
CMAKE_CXX_COMPILER_AR:FILEPATH=/usr/bin/gcc-ar-11

//A wrapper around 'ranlib' adding the appropriate '--plugin' option
// for the GCC compiler
CMAKE_CXX_COMPILER_RANLIB:FILEPATH=/usr/bin/gcc-ranlib-11

//Flags used by the CXX compiler during all build types.
CMAKE_CXX_FLAGS:STRING=-fPIC

//Flags used by the CXX compiler during DEBUG builds.
CMAKE_CXX_FLAGS_DEBUG:STRING=-g

//Flags used by the CXX compiler during MINSIZEREL builds.
CMAKE_CXX_FLAGS_MINSIZEREL:STRING=-Os -DNDEBUG

//Flags used by the CXX compiler during RELEASE builds.
CMAKE_CXX_FLAGS_RELEASE:STRING=-O3 -DNDEBUG

//Flags used by the CXX compiler during RELWITHDEBINFO builds.
CMAKE_CXX_FLAGS_RELWITHDEBINFO:STRING=-O2 -g -DNDEBUG

//C compiler
CMAKE_C_COMPILER:FILEPATH=/usr/bin/cc

//A wrapper around 'ar' adding the appropriate '--plugin' option
// for the GCC compiler
CMAKE_C_COMPILER_AR:FILEPATH=/usr/bin/gcc-ar-11

//A wrapper around 'ranlib' adding the appropriate '--plugin' option
// for the GCC compiler
CMAKE_C_COMPILER_RANLIB:FILEPATH=/usr/bin/gcc-ranlib-11

//Flags used by the C compiler during all build types.
CMAKE_C_FLAGS:STRING=-fPIC

//Flags used by the C compiler during DEBUG builds.
CMAKE_C_FLAGS_DEBUG:STRING=-g

//Flags used by the C compiler during MINSIZEREL builds.
CMAKE_C_FLAGS_MINSIZEREL:STRING=-Os -DNDEBUG

//Flags used by the C compiler during RELEASE builds.
CMAKE_C_FLAGS_RELEASE:STRING=-O3 -DNDEBUG

//Flags used by the C compiler during RELWITHDEBINFO builds.
CMAKE_C_FLAGS_RELWITHDEBINFO:STRING=-O2 -g -DNDEBUG

//No help, variable specified on the command line.
CMAKE_DISABLE_FIND_PACKAGE_ATSPI2:BOOL=ON

//No help, variable specified on the command line.
CMAKE_DISABLE_FIND_PACKAGE_CUPS:BOOL=ON

//No help, variable specified on the command line.
CMAKE_DISABLE_FIND_PACKAGE_DB2:BOOL=ON

//No help, variable specified on the command line.
CMAKE_DISABLE_FIND_PACKAGE_DirectFB:BOOL=ON

//No help, variable specified on the command line.
CMAKE_DISABLE_FIND_PACKAGE_EGL:UNINITIALIZED=OFF

//No help, variable specified on the command line.
CMAKE_DISABLE_FIND_PACKAGE_Fontconfig:UNINITIALIZED=OFF

//No help, variable specified on the command line.
CMAKE_DISABLE_FIND_PACKAGE_GLESv2:UNINITIALIZED=ON

//No help, variable specified on the command line.
CMAKE_DISABLE_FIND_PACKAGE_GLIB2:UNINITIALIZED=ON

//No help, variable specified on the command line.
CMAKE_DISABLE_FIND_PACKAGE_GSSAPI:BOOL=ON

//No help, variable specified on the command line.
CMAKE_DISABLE_FIND_PACKAGE_GTK3:UNINITIALIZED=ON

//No help, variable specified on the command line.
CMAKE_DISABLE_FIND_PACKAGE_ICU:UNINITIALIZED=OFF

//No help, variable specified on the command line.
CMAKE_DISABLE_FIND_PACKAGE_Interbase:BOOL=ON

//No help, variable specified on the command line.
CMAKE_DISABLE_FIND_PACKAGE_JPEG:UNINITIALIZED=OFF

//No help, variable specified on the command line.
CMAKE_DISABLE_FIND_PACKAGE_LTTngUST:BOOL=ON

//No help, variable specified on the command line.
CMAKE_DISABLE_FIND_PACKAGE_Libdrm:BOOL=ON

//No help, variable specified on the command line.
CMAKE_DISABLE_FIND_PACKAGE_Libinput:BOOL=ON

//No help, variable specified on the command line.
CMAKE_DISABLE_FIND_PACKAGE_Libproxy:BOOL=ON

//No help, variable specified on the command line.
CMAKE_DISABLE_FIND_PACKAGE_Libsystemd:BOOL=ON

//No help, variable specified on the command line.
CMAKE_DISABLE_FIND_PACKAGE_Libudev:BOOL=ON

//No help, variable specified on the command line.
CMAKE_DISABLE_FIND_PACKAGE_Mimer:BOOL=ON

//No help, variable specified on the command line.
CMAKE_DISABLE_FIND_PACKAGE_Mtdev:BOOL=ON

//No help, variable specified on the command line.
CMAKE_DISABLE_FIND_PACKAGE_MySQL:UNINITIALIZED=ON

//No help, variable specified on the command line.
CMAKE_DISABLE_FIND_PACKAGE_ODBC:UNINITIALIZED=ON

//No help, variable specified on the command line.
CMAKE_DISABLE_FIND_PACKAGE_Oracle:UNINITIALIZED=ON

//No help, variable specified on the command line.
CMAKE_DISABLE_FIND_PACKAGE_PNG:UNINITIALIZED=OFF

//No help, variable specified on the command line.
CMAKE_DISABLE_FIND_PACKAGE_PPS:BOOL=ON

//No help, variable specified on the command line.
CMAKE_DISABLE_FIND_PACKAGE_PostgreSQL:UNINITIALIZED=OFF

//No help, variable specified on the command line.
CMAKE_DISABLE_FIND_PACKAGE_SQLite3:UNINITIALIZED=OFF

//No help, variable specified on the command line.
...
Skipped 7425 lines
...
_OPENSSL_LDFLAGS_OTHER:INTERNAL=-pthread
_OPENSSL_LIBDIR:INTERNAL=/home/charlie/Documents/workspaces/workspace-vscode/vcpkg/installed/x64-linux/lib/pkgconfig/../../lib
_OPENSSL_LIBRARIES:INTERNAL=ssl;crypto;dl
_OPENSSL_LIBRARY_DIRS:INTERNAL=/home/charlie/Documents/workspaces/workspace-vscode/vcpkg/installed/x64-linux/lib/pkgconfig/../../lib
_OPENSSL_LIBS:INTERNAL=
_OPENSSL_LIBS_L:INTERNAL=
_OPENSSL_LIBS_OTHER:INTERNAL=
_OPENSSL_LIBS_PATHS:INTERNAL=
_OPENSSL_MODULE_NAME:INTERNAL=openssl
_OPENSSL_PREFIX:INTERNAL=/home/charlie/Documents/workspaces/workspace-vscode/vcpkg/installed/x64-linux/lib/pkgconfig/../..
_OPENSSL_STATIC_CFLAGS:INTERNAL=-I/home/charlie/Documents/workspaces/workspace-vscode/vcpkg/installed/x64-linux/lib/pkgconfig/../../include
_OPENSSL_STATIC_CFLAGS_I:INTERNAL=
_OPENSSL_STATIC_CFLAGS_OTHER:INTERNAL=
_OPENSSL_STATIC_INCLUDE_DIRS:INTERNAL=/home/charlie/Documents/workspaces/workspace-vscode/vcpkg/installed/x64-linux/lib/pkgconfig/../../include
_OPENSSL_STATIC_LDFLAGS:INTERNAL=-L/home/charlie/Documents/workspaces/workspace-vscode/vcpkg/installed/x64-linux/lib/pkgconfig/../../lib;-lssl;-lcrypto;-ldl;-pthread
_OPENSSL_STATIC_LDFLAGS_OTHER:INTERNAL=-pthread
_OPENSSL_STATIC_LIBDIR:INTERNAL=
_OPENSSL_STATIC_LIBRARIES:INTERNAL=ssl;crypto;dl
_OPENSSL_STATIC_LIBRARY_DIRS:INTERNAL=/home/charlie/Documents/workspaces/workspace-vscode/vcpkg/installed/x64-linux/lib/pkgconfig/../../lib
_OPENSSL_STATIC_LIBS:INTERNAL=
_OPENSSL_STATIC_LIBS_L:INTERNAL=
_OPENSSL_STATIC_LIBS_OTHER:INTERNAL=
_OPENSSL_STATIC_LIBS_PATHS:INTERNAL=
_OPENSSL_VERSION:INTERNAL=3.2.0
_OPENSSL_openssl_INCLUDEDIR:INTERNAL=
_OPENSSL_openssl_LIBDIR:INTERNAL=
_OPENSSL_openssl_PREFIX:INTERNAL=
_OPENSSL_openssl_VERSION:INTERNAL=
__pkg_config_arguments_Libb2:INTERNAL=IMPORTED_TARGET;libb2
__pkg_config_arguments_PC_DBUS1:INTERNAL=QUIET;dbus-1
__pkg_config_arguments_PC_EXPAT:INTERNAL=QUIET;expat
__pkg_config_arguments_PKG_EGL:INTERNAL=QUIET;egl
__pkg_config_arguments_PKG_FONTCONFIG:INTERNAL=QUIET;fontconfig
__pkg_config_arguments_PKG_X11_XCB:INTERNAL=QUIET;x11-xcb
__pkg_config_arguments_PKG_XCB_ATOM:INTERNAL=QUIET;xcb-atom
__pkg_config_arguments_PKG_XCB_AUX:INTERNAL=QUIET;xcb-aux
__pkg_config_arguments_PKG_XCB_COMPOSITE:INTERNAL=QUIET;xcb-composite
__pkg_config_arguments_PKG_XCB_CURSOR:INTERNAL=QUIET;xcb-cursor
__pkg_config_arguments_PKG_XCB_DAMAGE:INTERNAL=QUIET;xcb-damage
__pkg_config_arguments_PKG_XCB_DPMS:INTERNAL=QUIET;xcb-dpms
__pkg_config_arguments_PKG_XCB_DRI2:INTERNAL=QUIET;xcb-dri2
__pkg_config_arguments_PKG_XCB_DRI3:INTERNAL=QUIET;xcb-dri3
__pkg_config_arguments_PKG_XCB_EVENT:INTERNAL=QUIET;xcb-event
__pkg_config_arguments_PKG_XCB_EWMH:INTERNAL=QUIET;xcb-ewmh
__pkg_config_arguments_PKG_XCB_GLX:INTERNAL=QUIET;xcb-glx
__pkg_config_arguments_PKG_XCB_ICCCM:INTERNAL=QUIET;xcb-icccm
__pkg_config_arguments_PKG_XCB_IMAGE:INTERNAL=QUIET;xcb-image
__pkg_config_arguments_PKG_XCB_KEYSYMS:INTERNAL=QUIET;xcb-keysyms
__pkg_config_arguments_PKG_XCB_PRESENT:INTERNAL=QUIET;xcb-present
__pkg_config_arguments_PKG_XCB_RANDR:INTERNAL=QUIET;xcb-randr
__pkg_config_arguments_PKG_XCB_RECORD:INTERNAL=QUIET;xcb-record
__pkg_config_arguments_PKG_XCB_RENDER:INTERNAL=QUIET;xcb-render
__pkg_config_arguments_PKG_XCB_RENDERUTIL:INTERNAL=QUIET;xcb-renderutil
__pkg_config_arguments_PKG_XCB_RES:INTERNAL=QUIET;xcb-res
__pkg_config_arguments_PKG_XCB_SCREENSAVER:INTERNAL=QUIET;xcb-screensaver
__pkg_config_arguments_PKG_XCB_SHAPE:INTERNAL=QUIET;xcb-shape
__pkg_config_arguments_PKG_XCB_SHM:INTERNAL=QUIET;xcb-shm
__pkg_config_arguments_PKG_XCB_SYNC:INTERNAL=QUIET;xcb-sync
__pkg_config_arguments_PKG_XCB_UTIL:INTERNAL=QUIET;xcb-util
__pkg_config_arguments_PKG_XCB_XCB:INTERNAL=QUIET;xcb
__pkg_config_arguments_PKG_XCB_XF86DRI:INTERNAL=QUIET;xcb-xf86dri
__pkg_config_arguments_PKG_XCB_XFIXES:INTERNAL=QUIET;xcb-xfixes
__pkg_config_arguments_PKG_XCB_XINERAMA:INTERNAL=QUIET;xcb-xinerama
__pkg_config_arguments_PKG_XCB_XINPUT:INTERNAL=QUIET;xcb-xinput
__pkg_config_arguments_PKG_XCB_XKB:INTERNAL=QUIET;xcb-xkb
__pkg_config_arguments_PKG_XCB_XTEST:INTERNAL=QUIET;xcb-xtest
__pkg_config_arguments_PKG_XCB_XV:INTERNAL=QUIET;xcb-xv
__pkg_config_arguments_PKG_XCB_XVMC:INTERNAL=QUIET;xcb-xvmc
__pkg_config_arguments_PKG_XKB:INTERNAL=QUIET;xkbcommon
__pkg_config_arguments_XKB_COMMON_X11:INTERNAL=IMPORTED_TARGET;xkbcommon-x11>=0.4.1
__pkg_config_arguments_XRender:INTERNAL=IMPORTED_TARGET;xrender
__pkg_config_arguments__OPENSSL:INTERNAL=QUIET;openssl
__pkg_config_checked_Libb2:INTERNAL=1
__pkg_config_checked_PC_DBUS1:INTERNAL=1
__pkg_config_checked_PC_EXPAT:INTERNAL=1
__pkg_config_checked_PKG_EGL:INTERNAL=1
__pkg_config_checked_PKG_FONTCONFIG:INTERNAL=1
__pkg_config_checked_PKG_Wayland_Client:INTERNAL=1
__pkg_config_checked_PKG_Wayland_Cursor:INTERNAL=1
__pkg_config_checked_PKG_Wayland_Egl:INTERNAL=1
__pkg_config_checked_PKG_Wayland_Server:INTERNAL=1
__pkg_config_checked_PKG_X11_XCB:INTERNAL=1
__pkg_config_checked_PKG_XCB_ATOM:INTERNAL=1
__pkg_config_checked_PKG_XCB_AUX:INTERNAL=1
__pkg_config_checked_PKG_XCB_COMPOSITE:INTERNAL=1
__pkg_config_checked_PKG_XCB_CURSOR:INTERNAL=1
__pkg_config_checked_PKG_XCB_DAMAGE:INTERNAL=1
__pkg_config_checked_PKG_XCB_DPMS:INTERNAL=1
__pkg_config_checked_PKG_XCB_DRI2:INTERNAL=1
__pkg_config_checked_PKG_XCB_DRI3:INTERNAL=1
__pkg_config_checked_PKG_XCB_EVENT:INTERNAL=1
__pkg_config_checked_PKG_XCB_EWMH:INTERNAL=1
__pkg_config_checked_PKG_XCB_GLX:INTERNAL=1
__pkg_config_checked_PKG_XCB_ICCCM:INTERNAL=1
__pkg_config_checked_PKG_XCB_IMAGE:INTERNAL=1
__pkg_config_checked_PKG_XCB_KEYSYMS:INTERNAL=1
__pkg_config_checked_PKG_XCB_PRESENT:INTERNAL=1
__pkg_config_checked_PKG_XCB_RANDR:INTERNAL=1
__pkg_config_checked_PKG_XCB_RECORD:INTERNAL=1
__pkg_config_checked_PKG_XCB_RENDER:INTERNAL=1
__pkg_config_checked_PKG_XCB_RENDERUTIL:INTERNAL=1
__pkg_config_checked_PKG_XCB_RES:INTERNAL=1
__pkg_config_checked_PKG_XCB_SCREENSAVER:INTERNAL=1
__pkg_config_checked_PKG_XCB_SHAPE:INTERNAL=1
__pkg_config_checked_PKG_XCB_SHM:INTERNAL=1
__pkg_config_checked_PKG_XCB_SYNC:INTERNAL=1
__pkg_config_checked_PKG_XCB_UTIL:INTERNAL=1
__pkg_config_checked_PKG_XCB_XCB:INTERNAL=1
__pkg_config_checked_PKG_XCB_XEVIE:INTERNAL=1
__pkg_config_checked_PKG_XCB_XF86DRI:INTERNAL=1
__pkg_config_checked_PKG_XCB_XFIXES:INTERNAL=1
__pkg_config_checked_PKG_XCB_XINERAMA:INTERNAL=1
__pkg_config_checked_PKG_XCB_XINPUT:INTERNAL=1
__pkg_config_checked_PKG_XCB_XKB:INTERNAL=1
__pkg_config_checked_PKG_XCB_XPRINT:INTERNAL=1
__pkg_config_checked_PKG_XCB_XTEST:INTERNAL=1
__pkg_config_checked_PKG_XCB_XV:INTERNAL=1
__pkg_config_checked_PKG_XCB_XVMC:INTERNAL=1
__pkg_config_checked_PKG_XKB:INTERNAL=1
__pkg_config_checked_XKB_COMMON_X11:INTERNAL=1
__pkg_config_checked_XRender:INTERNAL=1
__pkg_config_checked__OPENSSL:INTERNAL=1
//Result of TRY_COMPILE
_arch_result:INTERNAL=TRUE
//ADVANCED property for variable: pkgcfg_lib_Libb2_b2
pkgcfg_lib_Libb2_b2-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PC_DBUS1_dbus-1
pkgcfg_lib_PC_DBUS1_dbus-1-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PC_EXPAT_expat
pkgcfg_lib_PC_EXPAT_expat-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PC_EXPAT_m
pkgcfg_lib_PC_EXPAT_m-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_EGL_EGL
pkgcfg_lib_PKG_EGL_EGL-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_FONTCONFIG_brotlicommon
pkgcfg_lib_PKG_FONTCONFIG_brotlicommon-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_FONTCONFIG_brotlidec
pkgcfg_lib_PKG_FONTCONFIG_brotlidec-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_FONTCONFIG_bz2
pkgcfg_lib_PKG_FONTCONFIG_bz2-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_FONTCONFIG_expat
pkgcfg_lib_PKG_FONTCONFIG_expat-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_FONTCONFIG_fontconfig
pkgcfg_lib_PKG_FONTCONFIG_fontconfig-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_FONTCONFIG_freetype
pkgcfg_lib_PKG_FONTCONFIG_freetype-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_FONTCONFIG_m
pkgcfg_lib_PKG_FONTCONFIG_m-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_FONTCONFIG_png16
pkgcfg_lib_PKG_FONTCONFIG_png16-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_FONTCONFIG_z
pkgcfg_lib_PKG_FONTCONFIG_z-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_X11_XCB_X11
pkgcfg_lib_PKG_X11_XCB_X11-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_X11_XCB_X11-xcb
pkgcfg_lib_PKG_X11_XCB_X11-xcb-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_X11_XCB_xcb
pkgcfg_lib_PKG_X11_XCB_xcb-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_XCB_ATOM_xcb
pkgcfg_lib_PKG_XCB_ATOM_xcb-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_XCB_ATOM_xcb-util
pkgcfg_lib_PKG_XCB_ATOM_xcb-util-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_XCB_AUX_xcb
pkgcfg_lib_PKG_XCB_AUX_xcb-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_XCB_AUX_xcb-util
pkgcfg_lib_PKG_XCB_AUX_xcb-util-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_XCB_COMPOSITE_xcb-composite
pkgcfg_lib_PKG_XCB_COMPOSITE_xcb-composite-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_XCB_CURSOR_xcb
pkgcfg_lib_PKG_XCB_CURSOR_xcb-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_XCB_CURSOR_xcb-cursor
pkgcfg_lib_PKG_XCB_CURSOR_xcb-cursor-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_XCB_DAMAGE_xcb-damage
pkgcfg_lib_PKG_XCB_DAMAGE_xcb-damage-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_XCB_DPMS_xcb-dpms
pkgcfg_lib_PKG_XCB_DPMS_xcb-dpms-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_XCB_DRI2_xcb-dri2
pkgcfg_lib_PKG_XCB_DRI2_xcb-dri2-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_XCB_DRI3_xcb-dri3
pkgcfg_lib_PKG_XCB_DRI3_xcb-dri3-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_XCB_EVENT_xcb
pkgcfg_lib_PKG_XCB_EVENT_xcb-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_XCB_EVENT_xcb-util
pkgcfg_lib_PKG_XCB_EVENT_xcb-util-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_XCB_EWMH_xcb
pkgcfg_lib_PKG_XCB_EWMH_xcb-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_XCB_EWMH_xcb-ewmh
pkgcfg_lib_PKG_XCB_EWMH_xcb-ewmh-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_XCB_GLX_xcb-glx
pkgcfg_lib_PKG_XCB_GLX_xcb-glx-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_XCB_ICCCM_xcb
pkgcfg_lib_PKG_XCB_ICCCM_xcb-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_XCB_ICCCM_xcb-icccm
pkgcfg_lib_PKG_XCB_ICCCM_xcb-icccm-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_XCB_IMAGE_xcb
pkgcfg_lib_PKG_XCB_IMAGE_xcb-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_XCB_IMAGE_xcb-image
pkgcfg_lib_PKG_XCB_IMAGE_xcb-image-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_XCB_IMAGE_xcb-shm
pkgcfg_lib_PKG_XCB_IMAGE_xcb-shm-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_XCB_KEYSYMS_xcb
pkgcfg_lib_PKG_XCB_KEYSYMS_xcb-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_XCB_KEYSYMS_xcb-keysyms
pkgcfg_lib_PKG_XCB_KEYSYMS_xcb-keysyms-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_XCB_PRESENT_xcb-present
pkgcfg_lib_PKG_XCB_PRESENT_xcb-present-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_XCB_RANDR_xcb-randr
pkgcfg_lib_PKG_XCB_RANDR_xcb-randr-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_XCB_RECORD_xcb-record
pkgcfg_lib_PKG_XCB_RECORD_xcb-record-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_XCB_RENDERUTIL_xcb
pkgcfg_lib_PKG_XCB_RENDERUTIL_xcb-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_XCB_RENDERUTIL_xcb-render
pkgcfg_lib_PKG_XCB_RENDERUTIL_xcb-render-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_XCB_RENDERUTIL_xcb-render-util
pkgcfg_lib_PKG_XCB_RENDERUTIL_xcb-render-util-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_XCB_RENDER_xcb-render
pkgcfg_lib_PKG_XCB_RENDER_xcb-render-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_XCB_RES_xcb-res
pkgcfg_lib_PKG_XCB_RES_xcb-res-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_XCB_SCREENSAVER_xcb-screensaver
pkgcfg_lib_PKG_XCB_SCREENSAVER_xcb-screensaver-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_XCB_SHAPE_xcb-shape
pkgcfg_lib_PKG_XCB_SHAPE_xcb-shape-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_XCB_SHM_xcb-shm
pkgcfg_lib_PKG_XCB_SHM_xcb-shm-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_XCB_SYNC_xcb-sync
pkgcfg_lib_PKG_XCB_SYNC_xcb-sync-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_XCB_UTIL_xcb
pkgcfg_lib_PKG_XCB_UTIL_xcb-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_XCB_UTIL_xcb-util
pkgcfg_lib_PKG_XCB_UTIL_xcb-util-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_XCB_XCB_xcb
pkgcfg_lib_PKG_XCB_XCB_xcb-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_XCB_XF86DRI_xcb-xf86dri
pkgcfg_lib_PKG_XCB_XF86DRI_xcb-xf86dri-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_XCB_XFIXES_xcb-xfixes
pkgcfg_lib_PKG_XCB_XFIXES_xcb-xfixes-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_XCB_XINERAMA_xcb-xinerama
pkgcfg_lib_PKG_XCB_XINERAMA_xcb-xinerama-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_XCB_XINPUT_xcb-xinput
pkgcfg_lib_PKG_XCB_XINPUT_xcb-xinput-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_XCB_XKB_xcb-xkb
pkgcfg_lib_PKG_XCB_XKB_xcb-xkb-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_XCB_XTEST_xcb-xtest
pkgcfg_lib_PKG_XCB_XTEST_xcb-xtest-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_XCB_XVMC_xcb-xvmc
pkgcfg_lib_PKG_XCB_XVMC_xcb-xvmc-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_XCB_XV_xcb-xv
pkgcfg_lib_PKG_XCB_XV_xcb-xv-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_XKB_xkbcommon
pkgcfg_lib_PKG_XKB_xkbcommon-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_XKB_COMMON_X11_xkbcommon
pkgcfg_lib_XKB_COMMON_X11_xkbcommon-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_XKB_COMMON_X11_xkbcommon-x11
pkgcfg_lib_XKB_COMMON_X11_xkbcommon-x11-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_XRender_X11
pkgcfg_lib_XRender_X11-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_XRender_Xrender
pkgcfg_lib_XRender_Xrender-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib__OPENSSL_crypto
pkgcfg_lib__OPENSSL_crypto-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib__OPENSSL_dl
pkgcfg_lib__OPENSSL_dl-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib__OPENSSL_ssl
pkgcfg_lib__OPENSSL_ssl-ADVANCED:INTERNAL=1
prefix_result:INTERNAL=/usr
```
</details>

Hello @MonicaLiu0311 , thank you for your time.

I did try those commands but didn't fix the issue:
```shell
$ ./vcpkg version
vcpkg package management program version 2023-12-12-1c9ec1978a6b0c2b39c9e9554a96e3e275f7556e

See LICENSE.txt for license information.
```

Remove command:
```shell
charlie@charlie-B660M:~/Documents/workspaces/workspace-vscode/vcpkg$ ./vcpkg remove opengl opengl-registry qtbase --recurse
The following packages are not installed:
    qtbase:x64-linux
The following packages will be removed:
  * egl:x64-linux
    opengl:x64-linux
    opengl-registry:x64-linux
warning: Additional packages (*) need to be removed to complete this operation.
Removing 1/3 egl:x64-linux
Removing 2/3 opengl:x64-linux
Removing 3/3 opengl-registry:x64-linux
```

Try to install again:
```shell
$ ./vcpkg install quazip --binarysource=clear
Computing installation plan...
The following packages will be built and installed:
  * egl:x64-linux@2022-12-04
  * opengl:x64-linux@2022-12-04#3
  * opengl-registry:x64-linux@2022-09-29#1
  * qt5compat[big-codecs,codecs,core,default-features,qml,textcodec]:x64-linux@6.6.1
  * qtbase[brotli,concurrent,core,dbus,default-features,doubleconversion,egl,fontconfig,freetype,gui,harfbuzz,icu,jpeg,network,opengl,openssl,pcre2,png,sql,sql-psql,sql-sqlite,testlib,thread,widgets,xcb,xcb-xlib,xkb,xkbcommon-x11,xlib,xrender,zstd]:x64-linux@6.6.1#3
  * qtdeclarative:x64-linux@6.6.1
  * qtlanguageserver:x64-linux@6.6.1
  * qtshadertools:x64-linux@6.6.1
    quazip:x64-linux@1.4#1
Additional packages (*) will be modified to complete this operation.
Detecting compiler hash for triplet x64-linux...
Installing 1/9 opengl-registry:x64-linux@2022-09-29#1...
Building opengl-registry:x64-linux@2022-09-29#1...
-- Using cached KhronosGroup-OpenGL-Registry-5bae8738b23d06968e7c3a41308568120943ae77.tar.gz.
-- Cleaning sources at /home/charlie/Documents/workspaces/workspace-vscode/vcpkg/buildtrees/opengl-registry/src/120943ae77-d1a12ea691.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source /home/charlie/Documents/workspaces/workspace-vscode/vcpkg/downloads/KhronosGroup-OpenGL-Registry-5bae8738b23d06968e7c3a41308568120943ae77.tar.gz
-- Using source at /home/charlie/Documents/workspaces/workspace-vscode/vcpkg/buildtrees/opengl-registry/src/120943ae77-d1a12ea691.clean
-- Installing: /home/charlie/Documents/workspaces/workspace-vscode/vcpkg/packages/opengl-registry_x64-linux/share/opengl-registry/copyright
-- Performing post-build validation
Elapsed time to handle opengl-registry:x64-linux: 1.1 s
opengl-registry:x64-linux package ABI: 4c17d3d7facbf7a3f87ea26c0095e1c13be1f2891a1f2ee87d6165a26600fcea
Installing 2/9 opengl:x64-linux@2022-12-04#3...
Building opengl:x64-linux@2022-12-04#3...
-- Installing: /home/charlie/Documents/workspaces/workspace-vscode/vcpkg/packages/opengl_x64-linux/share/opengl/usage
-- Performing post-build validation
Elapsed time to handle opengl:x64-linux: 10.4 ms
opengl:x64-linux package ABI: 0db3fc3e4a385638ac89ab035b2737fef159ac43d844fa2d2177dd37b038a713
Installing 3/9 egl:x64-linux@2022-12-04...
Building egl:x64-linux@2022-12-04...
-- Performing post-build validation
Elapsed time to handle egl:x64-linux: 10.1 ms
egl:x64-linux package ABI: 134964044cadf7c2142efe7a91e9790ff0f61d13bc0ae9da7d22efbcf2e19d51
Installing 4/9 qtbase[brotli,concurrent,core,dbus,default-features,doubleconversion,egl,fontconfig,freetype,gui,harfbuzz,icu,jpeg,network,opengl,openssl,pcre2,png,sql,sql-psql,sql-sqlite,testlib,thread,widgets,xcb,xcb-xlib,xkb,xkbcommon-x11,xlib,xrender,zstd]:x64-linux@6.6.1#3...
Building qtbase[brotli,concurrent,core,dbus,default-features,doubleconversion,egl,fontconfig,freetype,gui,harfbuzz,icu,jpeg,network,opengl,openssl,pcre2,png,sql,sql-psql,sql-sqlite,testlib,thread,widgets,xcb,xcb-xlib,xkb,xkbcommon-x11,xlib,xrender,zstd]:x64-linux@6.6.1#3...
CMake Warning at ports/qtbase/portfile.cmake:39 (message):
  qtbase currently requires packages from the system package manager.  They
  can be installed on Ubuntu systems via sudo apt-get install '^libxcb.*-dev'
  libx11-xcb-dev libglu1-mesa-dev libxrender-dev libxi-dev libxkbcommon-dev
  libxkbcommon-x11-dev.
Call Stack (most recent call first):
  scripts/ports.cmake:170 (include)


-- Using cached qtbase-everywhere-src-6.6.1.tar.xz.
-- Cleaning sources at /home/charlie/Documents/workspaces/workspace-vscode/vcpkg/buildtrees/qtbase/src/here-src-6-8b9bc1db6f.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source /home/charlie/Documents/workspaces/workspace-vscode/vcpkg/downloads/qtbase-everywhere-src-6.6.1.tar.xz
-- Applying patch allow_outside_prefix.patch
-- Applying patch config_install.patch
-- Applying patch fix_cmake_build.patch
-- Applying patch fix_cmake_build_type.patch
-- Applying patch harfbuzz.patch
-- Applying patch fix_egl.patch
-- Applying patch fix_egl_2.patch
-- Applying patch installed_dir.patch
-- Applying patch GLIB2-static.patch
-- Applying patch clang-cl_source_location.patch
-- Applying patch clang-cl_QGADGET_fix.diff
-- Applying patch fix-host-aliasing.patch
-- Applying patch dont_force_cmakecache_latest.patch
-- Using source at /home/charlie/Documents/workspaces/workspace-vscode/vcpkg/buildtrees/qtbase/src/here-src-6-8b9bc1db6f.clean
-- Configuring x64-linux
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:112 (message):
    Command failed: /home/charlie/Documents/workspaces/workspace-vscode/vcpkg/downloads/tools/ninja/1.10.2-linux/ninja -v
    Working Directory: /home/charlie/Documents/workspaces/workspace-vscode/vcpkg/buildtrees/qtbase/x64-linux-rel/vcpkg-parallel-configure
    Error code: 1
    See logs for more information:
      /home/charlie/Documents/workspaces/workspace-vscode/vcpkg/buildtrees/qtbase/config-x64-linux-dbg-CMakeCache.txt.log
      /home/charlie/Documents/workspaces/workspace-vscode/vcpkg/buildtrees/qtbase/config-x64-linux-rel-CMakeCache.txt.log
      /home/charlie/Documents/workspaces/workspace-vscode/vcpkg/buildtrees/qtbase/config-x64-linux-out.log

Call Stack (most recent call first):
  installed/x64-linux/share/vcpkg-cmake/vcpkg_cmake_configure.cmake:252 (vcpkg_execute_required_process)
  ports/qtbase/cmake/qt_install_submodule.cmake:147 (vcpkg_cmake_configure)
  ports/qtbase/cmake/qt_install_submodule.cmake:341 (qt_cmake_configure)
  ports/qtbase/portfile.cmake:293 (qt_install_submodule)
  scripts/ports.cmake:170 (include)


error: building qtbase:x64-linux failed with: BUILD_FAILED
Elapsed time to handle qtbase:x64-linux: 11 s
Please ensure you're using the latest port files with `git pull` and `vcpkg update`.
Then check for known issues at:
  https://github.com/microsoft/vcpkg/issues?q=is%3Aissue+is%3Aopen+in%3Atitle+qtbase
You can submit a new issue at:
  https://github.com/microsoft/vcpkg/issues/new?title=[qtbase]+Build+error+on+x64-linux&body=Copy+issue+body+from+%2Fhome%2Fcharlie%2FDocuments%2Fworkspaces%2Fworkspace-vscode%2Fvcpkg%2Finstalled%2Fvcpkg%2Fissue_body.md
```

You will find _issue_body_ file attached
[issue_body.md](https://github.com/microsoft/vcpkg/files/13750645/issue_body.md)

--- a/ports/qtbase/portfile.cmake
+++ b/ports/qtbase/portfile.cmake
@@ -39,7 +39,7 @@ if(VCPKG_TARGET_IS_LINUX)
     message(WARNING "qtbase currently requires packages from the system package manager. "
     "They can be installed on Ubuntu systems via sudo apt-get install " 
     "'^libxcb.*-dev' libx11-xcb-dev libglu1-mesa-dev libxrender-dev libxi-dev libxkbcommon-dev "
-    "libxkbcommon-x11-dev.")
+    "libxkbcommon-x11-dev libegl1-mesa-dev.")
 endif()
 
 # Features can be found via searching for qt_feature in all configure.cmake files in the source:

--- a/ports/qtbase/vcpkg.json
+++ b/ports/qtbase/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "qtbase",
   "version": "6.6.1",
-  "port-version": 4,
+  "port-version": 5,
   "description": "Qt Application Framework Base Module. Includes Core, GUI, Widgets, Networking, SQL, Concurrent and other essential qt components.",
   "homepage": "https://www.qt.io/",
   "license": null,

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -7118,7 +7118,7 @@
     },
     "qtbase": {
       "baseline": "6.6.1",
-      "port-version": 4
+      "port-version": 5
     },
     "qtcharts": {
       "baseline": "6.6.1",

--- a/versions/q-/qtbase.json
+++ b/versions/q-/qtbase.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "1b947014e92b75d7bf37427fc4a9ce3fd463af2c",
+      "version": "6.6.1",
+      "port-version": 5
+    },
+    {
       "git-tree": "7e1dba7994d7d228fee34f8f5b646e7d9aff62ac",
       "version": "6.6.1",
       "port-version": 4

